code/modules/clothing/suits/ego_gear/aleph.dm

```diff
@@ -15,7 +15,7 @@
 	name = "justitia"
 	desc = "A black, bandaged coat with golden linings covering it."
 	icon_state = "justitia"
-	armor = list(RED_DAMAGE = 60, WHITE_DAMAGE = 60, BLACK_DAMAGE = 60, PALE_DAMAGE = 75)
+	armor = list(RED_DAMAGE = 60, WHITE_DAMAGE = 60, BLACK_DAMAGE = 50, PALE_DAMAGE = 75)
```

Comment:
Stealth nerf smh smh.

Reply:
Funnily enough, in LC it is 0.5/0.5/0.5/0.5

Reply:
Then it would just be Worse SOAS.
![image](https://user-images.githubusercontent.com/77302679/185758361-d309f8ca-242f-4454-942f-bcff8ddef8e2.png)
This is the current spreadsheet of stats.
We have to keep all armor balanced in some way. Just keeping numbers exactly as Lobotomy corp would NOT work.

Now while Raw stat total doesn't mean anything, having it be 60/60/60/75 really makes it edge out a lot of other Aleph armor. Black armor is important, especially late game, and having a -10 on one of the only good pale armors (The other being blooming, which you need pale armor) stops it from being EVEN BETTER, as armor you get from a WAW.

Reply:
And, Spoiler, Pale armor is BAD. It is only worth getting for one abnormality, the rose. Otherwise it's whatever.
Having it being 60/60/50/75 gives it a niche over SOAS, for when you don't have Blue Star or need that extra 35 pale for whatever reason